schema/schema_template.py

```diff
@@ -79,27 +80,27 @@ class Instruments(Base):
 
 class PhotometryFilters(Base):
     """
-    ORM for filter table.
-    This stores relationships between filters and instruments,
-    telescopes, as well as wavelength and width
+    ORM for PhotometryFilters table.
+    This stores information about the filters as well as wavelength and width
     """
 
     __tablename__ = "PhotometryFilters"
-    band = Column(String(30), primary_key=True, nullable=False)
-    # of the form instrument.filter (see SVO)
-    instrument = Column(
-        String(30),
-        ForeignKey("Instruments.instrument", onupdate="cascade"),
-        primary_key=True,
-    )
-    telescope = Column(
-        String(30),
-        ForeignKey("Telescopes.telescope", onupdate="cascade"),
-        primary_key=True,
-    )
+    band = Column(String(30), primary_key=True, nullable=False)  # of the form instrument.filter (see SVO)
     effective_wavelength = Column(Float, nullable=False)
     width = Column(Float)
 
+    @validates("band")
+    def validate_band(self, key, value):
+        if "." not in value:
+            raise ValueError("Band name must be of the form instrument.filter")
+        return value
+
+    @validates("effective_wavelength")
+    def validate_wavelength(self, key, value):
+        if value is not None and value < 0:
```

Comment:
Checking via a test, since I thought a None value would fail since it's `nullable=False` but I think that aspect only gets checked if attempting to ingest into the database...

Reply:
I've updated it, but the validation is limited: if a user does not supply a value at all, the validation will not run. 
I think this is because it's not set as a primary key (which we don't want to do on a numeric field), but just a limitation of python. Regardless- it would get validated on an INSERT